editor/src/clj/editor/analytics.clj

```diff
@@ -311,37 +268,28 @@
 (defn enabled? []
   (some? @worker-atom))
 
-(defn set-uid! [^String uid]
-  {:pre [(or (nil? uid) (valid-uid? uid))]}
-  (swap! config-atom
-         (fn [config]
-           (let [config (or config (read-config! false))
-                 updated-config (assoc config :uid uid)]
-             (when (not= config updated-config)
-               (write-config! updated-config))
-             updated-config))))
-
 (defn track-event!
   ([^String category ^String action]
-   (append-event! {:t "event"
-                   :ec category
-                   :ea action}))
+   (append-event! {:name "select_content"
+                   :params {:content_type (str category "-" action)
+                            :engagement_time_msec "100"
```

Comment:
engagement_time_msec and session_id are required to get user tracking in Google Analytics.

We haven't really cared about engagement time previously, but if we really need to then we can try to measure it.